docs/docs/components/textinput.md

```diff
@@ -117,6 +114,11 @@ secureTextEntry: boolean = false;
 // Should spell checking be applied to contents?
 spellCheck: boolean = [value of autoCorrect];
 
+// iOS and Windows only property for controlling when the clear button should appear on the right side of the text view.
+// Default behavior is dependent on platform: equivalent to 'never' on iOS, and 'always' on Windows.
+// Button is hidden when text view is empty, regardless of the property value.
+clearButtonMode: 'never' | 'while-editing' | 'unless-editing' | 'always'
```

Comment:
Nit: missing a semicolon at the end.

Reply:
We could define another value 'default' and map that to the default value for each platform. Or we could just specify that an undefined value maps to the default behavior. I'm fine with either approach.

Reply:
Howard would prefer the default to be 'never', aligned with the S4L design that abhors the clear buttons.
I'm more inclined to give platform some "personality" choice..
Currently it is coded as/assumes "undefined means whatever platform chooses as the default"